autoload/verilog_systemverilog.vim

```diff
@@ -525,7 +525,7 @@ function! verilog_systemverilog#VerilogErrorFormat(...)
       echo "\n"
     endif
   else
-    let l_mode = a:2
+    let l:mode = a:2
```

Comment:
I will include this fix directly in my original commit.
Thanks for spotting it!